README.md

```diff
@@ -32,6 +32,28 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+We will be using external library called Requests (https://docs.python-requests.org/en/master/) to make the API requests and interpret the JSON response. You need to add the requests library to your list of poetry dependencies in pyproject.toml:
+```bash
+$ poetry add requests
+```
+
+```python-dotenv
+We will be using Python-dotenv to reads key-value pairs from a .env file for API Key, Token and Board ID.
+
+pip install python-dotenv
+```
```

Comment:
Now that you've run `poetry add requests`, it's tracked in your `pyproject.toml` file (take a look at the changes in that file) so future developers will pick it up for free when they run `poetry install`, and similarly you should find that `python-dotenv` is already installed as long as you're running the app with `poetry run flask run`